doc/nrf/applications.rst

```diff
@@ -0,0 +1,18 @@
+.. _applications:
+
+Applications
+############
+
+Applications consist of a fully integrated stack and interface drivers and libraries from the |NCS| and Zephyr.
+They implement a specific use case and can serve as a starting point for developing your product.
+
+Some applications come with a custom hardware.
```

Comment:
Some applications target custom hardware designed for a specific use-case, although they can also be run on different hardware like a generic development kit, albeit with limited functionality.